packages/components/psammead-figure/src/index.stories.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withReadme } from 'storybook-readme';
+import Image from '@bbc/psammead-image';
+import Readme from '../README.md';
+import Figure from './index';
+
+const imageAlt =
+  'Map of the UK displaying Syrian refugees and asylum seekers per 10000 population. Ranges from 0 to 17.';
+const imageSrc =
+  'https://ichef.bbci.co.uk/news/640/cpsprodpb/439A/production/_100960371_syrians_and_asylum_v2-nc.png';
+const imageWidth = 853;
+const imageHeight = 1067;
+
+storiesOf('Figure', module)
+  .addDecorator(withReadme(Readme))
+  .add('default', () => (
+    <Figure>
+      <Image
+        alt={imageAlt}
+        src={imageSrc}
+        width={imageWidth}
+        height={imageHeight}
```

Comment:
You'd likely not want to pass in height here, since it skews the image:
<img width="668" alt="screen shot 2018-12-19 at 13 34 01" src="https://user-images.githubusercontent.com/3028997/50223341-d549c180-0392-11e9-8e08-79bfb18c134a.png">